Description: Patch build to regenerate thrift
 interfaces at build time.
Author: Rich Lane <rlane@bigswitch.com>
Forwarded: not-needed

--- a/build.xml
+++ b/build.xml
@@ -81,7 +81,7 @@
         <mkdir dir="${thrift.out.dir}"/>
     </target>
 
-    <target name="compile" depends="init">
+    <target name="compile" depends="init,gen-thrift">
         <javac includeAntRuntime="false" 
 	       classpathref="classpath" 
 	       debug="true" 
